app/services/data/insert-child.js

```diff
@@ -0,0 +1,16 @@
+const config = require('../../../knexfile').extweb
+const knex = require('knex')(config)
+const AboutChild = require('../domain/about-child')
+
+module.exports = function (claimId, aboutChild) {
+  if (!(aboutChild instanceof AboutChild)) {
+    throw new Error('Provided object is not an instance of the expected class')
+  }
+
+  return knex('ClaimChild').insert({
+    ClaimId: claimId,
+    Name: aboutChild.childName || null,
```

Comment:
remove null default, if gets here with invalid data we should get persistence error rather than persist invalid